app/src/main/java/com/example/nehal/test/Register.java

```diff
@@ -0,0 +1,62 @@
+package com.example.nehal.test;
+
+import android.app.ProgressDialog;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+
+public class Register extends AppCompatActivity {
+
+    private EditText etEmail;
+    private EditText etPassword;
+    private FirebaseAuth firebaseAuth;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_register);
+        etEmail = (EditText) findViewById(R.id.etEmail);
+        etPassword= (EditText) findViewById(R.id.etPassword);
+        EditText etCnfPassword = (EditText) findViewById(R.id.etCnfPassword);
+        firebaseAuth = FirebaseAuth.getInstance();
+        String pwd = etPassword.getText().toString();
+        String conPwd = etCnfPassword.getText().toString();
+
+        if (!pwd.equals(" ") && !conPwd.equals(" ") && !pwd.equals(conPwd)){
```

Comment:
This is not the right way to test if the password is empty. See this http://stackoverflow.com/questions/3598770/java-check-whether-a-string-is-not-null-and-not-empty.